### ISSUE TITLE
Option to only let logged in users vote

### ISSUE THREAD

**Issue by duenni** [Taggic/rater#4]
Hi,

it would be nice to have an option set so that only logged in users are able to vote. Also if a logged in user votes twice only the last vote should be counted.
Thanks!

**Comment by guiohm** [comment]
I agree with that. I'm might take a look as I'm in real need of this feature.
